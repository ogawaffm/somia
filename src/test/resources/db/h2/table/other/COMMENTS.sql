DROP TABLE IF EXISTS TEST.COMMENTS
/
CREATE TABLE TEST.COMMENTS
(
    COMMENT_COL VARCHAR(10)
)
/
COMMENT ON TABLE TEST.COMMENTS IS 'This is a table comment'
/
COMMENT ON COLUMN TEST.COMMENTS.COMMENT_COL IS 'This is a column comment'
