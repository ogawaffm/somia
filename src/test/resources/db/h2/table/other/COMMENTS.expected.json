{
  "name": "COMMENTS",
  "catalogName": "TEST",
  "schemaName": "TEST",
  "typeName": "BASE TABLE",
  "comment": "This is a table comment",
  "columns": [
    {
      "name": "COMMENT_COL",
      "type": {
        "name": "CHARACTER VARYING",
        "typeNumber": 12,
        "className": "java.lang.String",
        "precision": 10,
        "scale": 0,
        "isNullable": true,
        "isAutoIncrement": false,
        "numericPrecisionRadix": 0,
        "isCaseSensitive": false,
        "isSearchable": true,
        "isSigned": false,
        "isCurrency": false,
        "columnDisplaySize": 10,
        "baseType": null
      },
      "comment": "This is a column comment",
      "collation": "",
      "defaultDefinition": ""
    }
  ],
  "indexes": [],
  "collation": "",
  "primaryKey": null,
  "bestRowIdentifier": null
}